lib/delivery_boy/config.rb

```diff
@@ -51,6 +51,7 @@ class Config < KingKonf::Config
     string :sasl_scram_username
     string :sasl_scram_password
     string :sasl_scram_mechanism
+    string :sasl_oauth_token_provider
```

Comment:
The example in README sets an object here, not a string. I don't understand what the value is supposed to be.